tests/unittest/resources/data/kubeconfig.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
```

Comment:
I don't like this getting committed to git's memory full of test public keys and tokens. Please remove this file from the PR and generate one during test setup instead.